modin/logging/config.py

```diff
@@ -169,7 +169,7 @@ def configure_logging() -> None:
         "modin.logger.default",
         job_id,
         "trace",
-        LogLevel.INFO if LogMode.get() == "enable_api_only" else LogLevel.DEBUG,
```

Comment:
Are we losing the ability to configure the debug level?

Reply:
Yes the ability to configure would be lost, however  the lower-level functionality would be made debug level (instead of info) in a further PR #7184